pkg/envinfo/envinfo.go

```diff
@@ -320,12 +345,17 @@ const (
 	Push = "PUSH"
 	// PushDescription is the description of URL
 	PushDescription = "Push parameter is the action to write devfile commands to env.yaml"
+	// Link parameter
+	Link = "LINK"
```

Comment:
I dont think we need this

Reply:
It otherwise fails at https://github.com/openshift/odo/blob/master/pkg/envinfo/envinfo.go#L325-L332.

Reply:
I meant we don’t need this being exposed which includes the line u mentioned as well.

Reply:
Do you mean to get rid of the check happening here https://github.com/openshift/odo/blob/master/pkg/envinfo/envinfo.go#L171 and the bits required to get it done?

Reply:
No, but it’s ok for now. We can solve this in a separate PR